Footnotes: update docs (#829)
exadel-inc/esl: src/modules/esl-footnotes/README.md

The elements are interrelated and don't make sense on their own. This is because once the elements are added to the DOM, they establish the link between ESLFootnotes and each ESLNote.

ESLNote is a badge with a number or symbol. When hovered or clicked (user-defined behavior), it shows a tooltip with text information. Depending on the state of the ESLNote, it may display:
 - `0` - if the note is not associated with notes
 - `*` - if the note is linked to the footnotes but in a standalone mode
 - `number 1...` - ordinal index from footnotes if linked to the notes in normal mode

### ESLFootnotes Attributes | Properties:

- `scopeTarget` (string) - target element ([TraversingQuery](../esl-traversing-query/README.md) selector) to define scope of footnotes (`::parent` by default)

- `grouping` (string) - grouping note instances with identical content. Available options:
  -  `enable` - default, notes with identical content will be grouped
  -  `disable` - without grouping

- `back-to-note-label` (string) - a text label to show in the title on the return-to-note button (`Back to note` by default)

### ESLNote Attributes | Properties:

> - `linked` - read-only marker that exists only on the note linked with the footnotes

Reviewer: I think word "appears" would sound more correctly
```suggestion
- `linked` - read-only marker that appears only on the note linked with the footnotes
```

> - `standalone` - read-only marker indicates that the note remains independent of the notes even if it is linked with them (ignore mode, for example) and doesn't receive its ordinal index from the footnotes (the default * or a user-defined value is displayed)

Reviewer: ```suggestion
- `standalone` - read-only marker which indicates that the note remains independent from the notes even if it is linked with them (ignore mode, for example) and doesn't receive its ordinal index from the footnotes (the default * or a user-defined value is displayed)
```

> - `tooltip-shown` - read-only marker which exists when notes tooltip is shown

Reviewer: Same here. I think word "appears" instead of "exists" does sound more correctly
```suggestion
- `tooltip-shown` - read-only marker which appears when notes tooltip is shown
```

> - `container` - target to container element ([TraversingQuery](../esl-traversing-query/README.md) selector) to define bounds of tooltip visibility (window by default)

Reviewer 2: Maybe it makes sense to write the definition like in popup docs:
defines container element ([TraversingQuery](../esl-traversing-query/README.md) selector) to determinate bounds of tooltip visibility (window by default)

Author: Fixed! Thanks!

> - `container` - target to container element ([TraversingQuery](../esl-traversing-query/README.md) selector) to define bounds of tooltip visibility (window by default)

Reviewer 3: Or even better the same as @NastaLeo suggested:
```suggestion
- `container` - defines container element ([TraversingQuery](../esl-traversing-query/README.md) selector) to determinate bounds of tooltip visibility (window by default)
```

> - `ignore` - [MediaQuery](../esl-media-query/README.md) to specify device media conditions when footnotes must ignore current note. (`not all` by default)

Reviewer 4: extra cosmetic: if you add points, then you need to add them everywhere (here and below)
```suggestion
- `ignore` - [MediaQuery](../esl-media-query/README.md) to specify device media conditions when footnotes must ignore current note (`not all` by default)
```
